Portal/src/Datahub.Portal/Layout/BarebonesPublicLayout.razor

```diff
@@ -0,0 +1,26 @@
+@using MudBlazor.Utilities
+@using Datahub.Portal.Pages.Public
+@inherits LayoutComponentBase
+
+@attribute [AllowAnonymous]
+
+<MudThemeProvider Theme="@DatahubTheme.DefaultTheme"/>
+<MudDialogProvider/>
+<MudSnackbarProvider/>
+<MudPopoverProvider/>
+
+<MudLayout Style="@_backgroundStyle">
+    @Body
+</MudLayout>
+
+@code {
+
+    private readonly string _backgroundStyle = new StyleBuilder()
```

Comment:
Low priority, can we blur this to match other GC sites? It would be good for consistency.

Ex: https://open-science.canada.ca/

Reply:
Total tangent but it's gross how they load every image and programmatically add the `hide` class instead of just programmatically changing the source url...

![image](https://github.com/user-attachments/assets/809e4660-38f9-4068-8444-ea630bae0ed0)

Reply:
@gcalexk if you want you can just add something like a `filter: blur(4px)`